examples/moses-pln-synergy/scm/README.md

```diff
@@ -76,16 +76,204 @@ And search for the following
+      (LambdaLink
+         (TypedVariableLink
+            (VariableNode "$X")
+            (TypeNode "ConceptNode")
+         )
+         (AndLink
+            (EvaluationLink
+               (PredicateNode "take")
+               (ListLink
+                  (VariableNode "$X")
+                  (ConceptNode "treatment-1")
+               )
+            )
+            (EvaluationLink (stv 1 0.99999982)
+               (PredicateNode "contain")
+               (ListLink
+                  (ConceptNode "treatment-1")
+                  (ConceptNode "compound-A")
+               )
+            )
+         )
+      )
+      (LambdaLink
+         (TypedVariableLink
+            (VariableNode "$X")
+            (TypeNode "ConceptNode")
+         )
+         (EvaluationLink
+            (PredicateNode "take")
+            (ListLink
+               (VariableNode "$X")
+               (ConceptNode "compound-A")
+            )
+         )
+      )
+   )
+...
+scheme@(guile-user)> (cog-bind and-lambda-distribution-rule)
+...
+   (AndLink
+      (LambdaLink
+         (TypedVariableLink
+            (VariableNode "$X")
+            (TypeNode "ConceptNode")
+         )
+         (EvaluationLink
+            (PredicateNode "take")
+            (ListLink
+               (VariableNode "$X")
+               (ConceptNode "treatment-1")
+            )
+         )
+      )
+      (LambdaLink
+         (TypedVariableLink
+            (VariableNode "$X")
+            (TypeNode "ConceptNode")
+         )
+         (EvaluationLink (stv 1 0.99999982)
+            (PredicateNode "contain")
+            (ListLink
+               (ConceptNode "treatment-1")
+               (ConceptNode "compound-A")
+            )
+         )
+      )
+   )
+...
+scheme@(guile-user)> (cog-bind lambda-grounded-construction-rule)
+$5 = (SetLink
+   (LambdaLink (stv 1 0.99999982)
+      (TypedVariableLink
+         (VariableNode "$X")
+         (TypeNode "ConceptNode")
+      )
+      (EvaluationLink (stv 1 0.99999982)
+         (PredicateNode "contain")
+         (ListLink
+            (ConceptNode "treatment-1")
+            (ConceptNode "compound-A")
+         )
+      )
    )
 )
 
+scheme@(guile-user)> (cog-bind implication-construction-rule)
+...
+   (ImplicationLink (stv 1 0.99999982)
+      (LambdaLink
+         (TypedVariableLink
+            (VariableNode "$X")
+            (TypeNode "ConceptNode")
+         )
+         (EvaluationLink
+            (PredicateNode "take")
+            (ListLink
+               (VariableNode "$X")
+               (ConceptNode "treatment-1")
+            )
+         )
+      )
+      (LambdaLink (stv 1 0.99999982)
+         (TypedVariableLink
+            (VariableNode "$X")
+            (TypeNode "ConceptNode")
+         )
+         (EvaluationLink (stv 1 0.99999982)
+            (PredicateNode "contain")
+            (ListLink
+               (ConceptNode "treatment-1")
+               (ConceptNode "compound-A")
+            )
+         )
```

Comment:
That is exactly what it is saying :-)

Since treatment-1 does contain compound-1, we can infer that anything implies that treatment-1 contains compound-1.